airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -311,6 +349,7 @@ class Projects(JiraStream):
 
     extract_field = "values"
     use_cache = True
+    is_resumable = False
```

Comment:
@brianjlai Could you please advise me with rfr?

I updated jira to v^3 airbyte-cdk and faced with the issue when python streams stuck because `next_slice` is never None to exit `while` loop [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/core.py#L177). 
`next_slice={}`
When I'm setting `is_resumable = False` stream works as expected.

What am I missing here?

Reply:
@brianjlai any insight on this one?